playground/src/playground.scss

```diff
@@ -0,0 +1,3 @@
+#playground_preview * {
```

Comment:
what about multiple playgrounds I think we will have multiple element with the same id.
What about to have `initial` instead of `unset` ?

Reply:
> what about multiple playgrounds I think we will have multiple element with the same id.
I know, but it shouldn't be an issue. The trouble is that it doesn't work with `class` instead of `id`, it's not specific enough.

> What about to have `initial` instead of `unset` ?
That sounds good and it works. Changed.

Reply:
ok